tmva/sofie/test/CMakeLists.txt

```diff
@@ -16,34 +16,29 @@ if (NOT ONNX_MODELS_DIR)
   set(ONNX_MODELS_DIR input_models)
 endif()
 
-protobuf_generate_cpp(PROTO_SRCS PROTO_HDRS "${SOFIE_PARSERS_DIR}/onnx_proto3")
-set_source_files_properties(${PROTO_SRCS} ${PROTO_HDRS} PROPERTIES GENERATED TRUE)
+#protobuf_generate_cpp(PROTO_SRCS PROTO_HDRS "${SOFIE_PARSERS_DIR}/onnx_proto3")
```

Comment:
LGTM!

Thanks for this change.

Can we remove these commented lines if not required anymore?

Reply:
Thank you ! I have removed these lines